index.html

```diff
@@ -173,6 +173,32 @@ <h2>Calls for implementation and feedback</h2>
 
 <p>When an addition is accepted at the “candidate” (stage 3) maturity level, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage and external feedback.
 
+<h2>Blocking Advancement</h2>
+
+<p>During the discussion of a proposal, at any stage, any aspect may be discussed, including those outside of the scope of its current stage. A proposal’s advancement may be blocked, however the committee does not have a concept of a rejected proposal.</p>
```

Comment:
```suggestion
<p>During the discussion of a proposal, at any stage, any aspect may be discussed, including those outside of the scope of its current stage. A proposal’s advancement may not have consensus, however the committee does not have a concept of a rejected proposal.</p>
```
also this isn't entirely true; the committee has rejected proposals - see https://github.com/tc39/proposals/blob/master/inactive-proposals.md for the ones labelled "rejected".

Reply:
We do not identify in the notes that these proposals are rejected. They are recorded as having something along the lines of ["not confident that all issues can be addressed"](https://github.com/tc39/notes/blob/master/meetings/2015-07/july-28.md#62-regexpescape) and ["Some members of the committee are strongly opposed to this particular syntax change"](https://github.com/tc39/notes/blob/8e8bfcbddcb29c09a10b0845a55af2e0d31b6f49/meetings/2017-01/jan-25.md#13iia-proposed-grammar-change-to-es-modules) or simply as "no advance". It looks like the entries in the inactive proposals list are opinionated rather than reflecting the notes. It would be more accurate to say that these are stalled, or withdrawn. We should probably be more strict in how we record things in the inactive proposals document. 

The closest we have to a rejection is [this one](https://github.com/tc39/notes/blob/7b2de881081abd34b02bc87bcdb662fd97555795/meetings/2016-07/jul-26.md#9iib-objectenumerablekeysvaluesentries) which was blocked as "Adding new features, in new browsers, to fix features in old browsers, is not what we should be doing." -- We _could_ say that this is a rejection. But this could equally be seen as not having consensus as per the last comment.

Reply:
It's also possible the notes aren't accurate there; for example, I was quite explicitly told that Error.isError was rejected when presenting it - I marked "withdrawn" since there was an alternative path to reach my goals. Similarly, Set/Map toJSON was also rejected.

In general, I think a lack of consensus for stage 1, when there's no path forward for the committee to want to spend further time on it, feels like "rejected" is the right word to me.